rpm/sensorfw-qt5-hybris.spec

```diff
@@ -37,18 +35,6 @@ Requires:   qt5-qtnetwork-devel
 %description devel
 Development headers for sensor framework daemon and libraries.
 
-
-%package configs
```

Comment:
I thought that was for the replacement configs package to obsolete this package?